Add hidden embedding parameters to the documentation (#15739)
metabase/metabase: docs/administration-guide/13-embedding.md

<iframe src="http://metabase.example.com/embed/dashboard/TOKEN" onload="iFrameResize({}, this)"></iframe>
```

### Additional parameters

> You can add to the embedding URL these additional parameters to affect the look and feel:
> - bordered: true or false. Adds a visible border to the embed
> - titled: true or false. Adds or removes the title to the embedding
> - theme: null or night. Shows the embedded iframe in dark mode 

Reviewer: ```suggestion
To change the look and feel of the embed, you can add additional parameters to the embedding URL:

- **bordered**: true or false. Adds a visible border to the embed.
- **titled**: true or false. Adds or removes the title to the embedding.
- **theme**: null or night. Shows the embedded iframe in dark mode. 
```

Can we also provide an example URL with these parameters set?

To learn more about embedding, check out these articles:

> - [How to use Metabase to deliver analytics to your customers](https://www.metabase.com/blog/external-facing-analytics/index.html).

Reviewer: ```suggestion
- [Embedding charts and dashboards](https://www.metabase.com/learn/developing-applications/advanced-metabase/embedding-charts-and-dashboards)
- [Four strategies for delivering customer-facing analytics with Metabase](https://www.metabase.com/learn/developing-applications/advanced-metabase/embedding-overview)
- [Embed Metabase in your app to deliver multi-tenant, self-service analytics](https://www.metabase.com/learn/developing-applications/advanced-metabase/multi-tenant-self-service-analytics.html)
```

> - [How to embed Metabase in your app to deliver multi-tenant, self-service analytics](https://www.metabase.com/blog/embedding/index.html).

Reviewer: ```suggestion
```
